bahmnicore-omod/src/main/resources/liquibase.xml

```diff
@@ -4807,6 +4807,111 @@
             INSERT INTO role_privilege(role, privilege) VALUES ("Appointments:ManageAppointments", "Create Teleconsultation");
         </sql>
     </changeSet>
+    <changeSet id="bahmni-core-202303201210" author="Parvathy, Rahul">
+        <preConditions onFail="MARK_RAN">
+            <not>
+                <sqlCheck expectedResult="0">
+                    SELECT count(concept_set_id) FROM concept_set WHERE concept_set = (SELECT concept_id FROM concept_name WHERE name = 'Dosing Units' AND concept_name_type = 'FULLY_SPECIFIED' AND locale_preferred=true)
+                    AND concept_id = (SELECT concept_id FROM concept_name WHERE name = 'Capsule(s)' AND concept_name_type = 'FULLY_SPECIFIED' AND locale_preferred=true);
+                </sqlCheck>
+            </not>
+            <not>
+                <sqlCheck expectedResult="1">
+                    SELECT count(concept_set_id) FROM concept_set WHERE concept_set = (SELECT concept_id FROM concept_name WHERE name = 'Dosing Units' AND concept_name_type = 'FULLY_SPECIFIED' AND locale_preferred=true)
+                    AND concept_id = (SELECT concept_id FROM concept_name WHERE name = 'Capsule(s)' AND concept_name_type = 'FULLY_SPECIFIED' AND locale_preferred=true);
+                </sqlCheck>
+            </not>
+        </preConditions>
+        <comment>Delete Capsule(s) from Dosing Unit concept set</comment>
+        <sql>
```

Comment:
@rahu1ramesh the intention is to remove duplicate Capsules(s) then we can mention the count as more than one as a precondition instead of  having 2 sql checks for same sql ?

Reply:
@binduak We checked if that was possible but expectedResult doesn't allow the use of '>' operator. sqlCheck executes an SQL string and checks the returned value against the value that expectedResult is set to.